tests/models/templates/test_csms_api.py

```diff
@@ -21,13 +24,335 @@
 from ...resources.test_trial_metadata import setup_user
 
 
-def mock_get_current_user(cidc_api, monkeypatch):
-    setup_user(cidc_api, monkeypatch)
+def test_detect_manifest_changes(cidc_api, clean_db, monkeypatch):
```

Comment:
So I think stylistically its better to have multiple tests, one for each scenario. I think if you breakup the change detection function into sub-functions you'll find it will be more natural.

Reply:
broken up in future commit